Makefile

```diff
@@ -268,11 +268,11 @@ format-frontend:
 ifdef check
 	$(PYTHON) ./bin/format.py --check $(if $(version:-=),--cfversion $(version)) ./frontend/catalyst/utils
 	black --check --verbose .
-	isort --check --diff .
+	isort --check --diff . --skip catalyst/mlir/llvm-project --skip frontend/catalyst/third_party
```

Comment:
Not sure if we need those. One thing I noticed between Catalyst and PennyLane is that we use different clang-format versions.

Reply:
EDIT: I think we can keep the skip here.

Reply:
I don't think skips should be configured here. Either in the code or in the pyproject.toml

Reply:
Good point, I forget about `pyproject` as we do not use it in Lightning (or at least until very recently).